data/remnant ships.txt

```diff
@@ -317,30 +469,101 @@ ship "Gascraft"
 		"shields" 800
 		"hull" 1500
 		"required crew" 1
-		"bunks" 1
-		"mass" 80
+		"bunks" 2
+		"mass" 78
 		"drag" 1.7
 		"heat dissipation" 0.9
 		"fuel capacity" 700
-		"cargo space" 8
-		"outfit space" 90
+		"energy capacity" 150
+		"ramscoop" 0.5
+		"cargo space" 9
```

Comment:
Likewise with the cargo increase; why?

Reply:
As with above, I don't see an end to the line of argument of "increased flexibility." The gascraft is intentionally a non-flexible ship to begin with.

Reply:
Alright, as above, ignore that and just go with "Because I'll need that space later."